statsmodels/sandbox/nonparametric/densityorthopoly.py

```diff
@@ -337,7 +337,7 @@ def evaluate(self, xeval, order=None):
         xeval = self._transform(xeval)
         if order is None:
             order = len(self.polys)
-        res = sum(c*p(xeval) for c, p in zip(self.coeffs, self.polys)[:order])
+        res = sum(c*p(xeval) for c, p in list(zip(self.coeffs, self.polys))[:order])
```

Comment:
Is the `list` necessary?  You're going to iterate over it and never see it again.